src/packages/server/index.js

```diff
@@ -26,9 +27,11 @@ class Server {
 
   router: Server$opts.router;
 
+  cors: Server$opts.server.cors;
+
   instance: HTTPServer;
 
-  constructor({ logger, router }: Server$opts) {
+  constructor({ logger, router, server: { cors } }: Server$opts) {
```

Comment:
I'm leaving open the idea that we eventually may have other types of configs under the server key—which I assumed was what you intended with the original config structure you suggested (and which I'm using here).